src/main/webapp/js/adminSearch.es6

```diff
@@ -53,6 +53,57 @@ $(document).ready(() => {
     $('.resetGoogleIdButton').click((e) => {
         e.stopPropagation();
     });
+
+    $('.openMailEditorWithDefaults').submit(function (e) {
+        e.preventDefault();
+        const receiveremailId = $('input[name=studentEmail]').val();
+        const subjectType = $(this).find('input.subjectType').val();
+        const studentName = $(this).find('input.studentName').val();
+        const courseName = $(this).find('input.courseName').val();
+        const Id = $(this).find('input.Id').val();
+        const uri = encodeURIComponent($(this).find('input.url').val());
```

Comment:
Any reason why `url` is used for the input class name while `uri` is used for the constant?